.commitlintrc.mjs

```diff
@@ -0,0 +1,41 @@
+import * as assert from "node:assert";
+
+let gimojis;
+try {
+    const res = await fetch(
+        "https://raw.githubusercontent.com/zeenix/gimoji/refs/heads/main/emojis.json",
+    );
+    assert.ok(res.ok);
+    const json = await res.json();
+    gimojis = json.gitmojis.map(({ emoji }) => emoji);
+} catch (cause) {
+    throw new Error("unable to fetch gimoji emojis.json", { cause });
+}
+
+export default {
+    defaultIgnores: true,
+    helpUrl: "https://github.com/z-galaxy/zbus/blob/main/CONTRIBUTING.md",
+    ignores: [
+        (commit) =>
+            commit.includes(
+                "This PR was generated by [Mend Renovate](https://mend.io/renovate/).",
```

Comment:
This reminds me, we need to also skip merge commits. Or, we could make the checks only run on PRs and not every push. That probably makes sense anyway since there is not much point in checking the commits/pushes to main branch (or any branch really).

Reply:
that's what the default ignores setting does: https://commitlint.js.org/reference/configuration.html#configuration-object-example

https://github.com/conventional-changelog/commitlint/blob/dddcc2834a24727ccb4cc63cdc4e7b7c06322e9b/%40commitlint/is-ignored/src/defaults.ts#L17